database.sql

```diff
@@ -78,8 +80,8 @@ CREATE TABLE `user` (
 -- Contenu de la table `user`
 --
 
-INSERT INTO `user` (`id`, `nickname`) VALUES
-(1, 'Trombone');
+INSERT INTO `user` (`nickname`, `email`, `password`) VALUES
+('Trombone', 'melissa.callejon@yahoo.fr', 'motdepasse');
 -- Structure de la table `user`
```

Comment:
je pense que cela ne sert pas à grand chose d'insérer un utilisateur en BDD avec un mot de passe en clair car tu as mis la fonction password_verify() dans le controleur UserController à la méthode login() (ce qui est très bien au passage) mais ne pourra jamais marcher sur la string `motdepasse` 😉, you see ?